ddocs/medic-client/views/reports_by_form_year_month_places/map.js

```diff
@@ -1,71 +1,24 @@
 // data record must adhere to property name of `month`
 function(doc) {
-  if (doc.type !== 'data_record') {
-    return;
+  if (doc.type === 'data_record' &&
```

Comment:
Interesting. I've see you doing this is several places. 
I find it clearer to do 
```
if (!condition) { return; }
return someStuff;
```
than
```
if (condition) { return someStuff; }
<implicit undefined return>
```
As a read or write the second version, I have to keep in mind what the "else return" is going to be, it's cognitive load, and if I forget it it's trouble.

Return-early principle : https://github.com/medic/medic-docs/blob/master/development/style-guide.md#return-early-from-functions (yes someone has quoted the styleguide)